CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 # Unreleased
 
+## TextView
+- Fixed bug on Android where setting `TextWrapping="NoWrap"` would force the `TextView` to be single line. New behavior is to instead allow the view to scroll horizontally instead of automatically wrapping the text.
+
 ## MultiDensityImageSource
 - Added native support, meaning it can be used by images inside a `NativeViewHost`.
 
-### Fuse.Reactive framework changes (Uno-level)
+## Fuse.Reactive framework changes (Uno-level)
```

Comment:
Remove this change? @kristianhasselknippe

Reply:
@nekronos: This was my change, and it's simply just a lazy version of a rebase. This change is already on master.